### PR TITLE
Fix width of highlight in S viewport

### PR DESCRIPTION
**Pull request by hqhhuang**
Bug/issue #, if applicable: 89800555

## Summary
Fix width of highlight in S viewport to avoid it being cut off.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] Added tests - NA
- [x] Ran `npm test`, and it succeeded
- [ ] Updated documentation if necessary - NA

### CODE REVIEW

**Comment by hqhhuang** [comment]
@swift-ci test

File: src/styles/core/_changes.scss [hunk 1]
```diff
@@ -78,7 +78,7 @@ $deprecated-dark-rounded-svg: $deprecated-svg !default;
 }
 
 @mixin change-highlight-end-spacing() {
-  padding-right: $change-icon-width + $change-highlight-horizontal-space-rem;
+  padding-right: $change-highlight-horizontal-space;
```
Comment (dobromir-hristov) [inline src/styles/core/_changes.scss:81]: I think this will make the icon step over the content.

**Review by dobromir-hristov — COMMENTED** [review]
I wonder if we add `box-sizing: border-box;` to the `.link-block.changed` would be enough to fix the problem 🤔

**Comment by hqhhuang** [comment]
@swift-ci test

**Review by dobromir-hristov — APPROVED** [review]
LGTM

**Comment by hqhhuang** [comment]
@swift-ci test

**Pull request merged**